### DISCUSSION TITLE
Feature Request: Add organization and default project labels

### DISCUSSION THREAD

**Discussion by nullndr** [community/community#28862] [Projects and Issues]
It would be great to have labels defined be the organization projects, or by the organization itself.

This would allow multiple repo to share common labels, deleting the developers to add them repo by repo.

It would also be great to have the same for personaly repositories.

**Comment by minherz** [comment]
I think there are already [default labels at organization level][1]. How do you see to implement a similar functionality on personal level? Where do you propose to setup these labels?

[1]: https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization

**Reply by nullndr** [reply]
Oh, I didn't know about the organization labels, great.

For personal repository it would be just a voice in the settings page.

**Comment by RallyTuning** [comment]
Yes, default labels for profiles will be great too. Considering every time i create a new repo (not in organization) i must delete unused labels and edit names and colors of the others.
So please, default labels for profile repos too! Thanks
